src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -352,9 +386,10 @@ public void setMigrationEndTimeAndStatus(String migrationId,
 						String zipFileStatus = zipFiles(type, httpContext,
 								zipFileName, title, webLinkUrl, contentAccessUrl,
```

Comment:
This method `zipFiles` is being called for the ZipMail migration as well and this returns a String STATUS not like status Object or OK/ERROR etc. So the current new format works for the Box migration/ resource Zip migration. But will not be appropriate for Mailzip Download.